server/app.js

```diff
@@ -4,36 +4,41 @@ import { join } from "path";
 import cookieParser from "cookie-parser";
 import logger from "morgan";
 import mongoose from "mongoose";
-const dotenv = require('dotenv');
+const busboy = require('connect-busboy');
+const busboyBodyParser = require('busboy-body-parser');
 
 import indexRouter from "./routes/index";
 import pingRouter from "./routes/ping";
-import auth from "./routes/auth";
-import verifyToken from "./routes/verifyToken";
+import authRouter from "./routes/auth";
+const shopIdRouter = require("./routes/shopId");
```

Comment:
Also this is not a standard naming convention for the files with NodeJS - https://stackoverflow.com/questions/18927298/node-js-project-naming-conventions-for-files-folders
OR
https://google.github.io/styleguide/jsguide.html#file-name

Filenames should probably be kebab-case
I would change it to even just shop though to be honest